.depcheckrc.json

```diff
@@ -7,16 +7,15 @@
     "@yarnpkg/types",
     "prettier-plugin-packagejson",
     "ts-node",
-    "@storybook/addon-actions",
-    "@storybook/react-webpack5",
-    "babel-loader",
-    "dependency-tree",
-    "eslint-plugin-node",
-    "sass",
-    "webpack",
     "@yarnpkg/core",
     "@yarnpkg/cli",
     "clipanion",
-    "@yarnpkg/fslib"
+    "@yarnpkg/fslib",
+    "@chromatic-com/storybook",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/addon-links",
+    "@storybook/blocks",
+    "@storybook/test"
```

Comment:
Adding new dependencies to depcheck file otherwise we get a linting error this is a known issue with the module template set up